### DISCUSSION TITLE
The new halloween effect on the new update of armoury crate Aura effects  5.7.3.0

### DISCUSSION THREAD

**Discussion by A-Madhu** [seerge/g-helper#1548] [Ideas]
New update to the armoury crate with aura sync that came a new effect aura sync which is time limited .... it's sick , can we get this effect on the next update of G

https://github.com/seerge/g-helper/assets/116909710/6270590f-0946-4004-990f-090d64e3d82e

![request](https://github.com/seerge/g-helper/assets/116909710/0bd0185a-6a7e-4623-9694-beaa81ac3b23)
.helper ? please <3 ???

**Comment by seerge** [comment]
@A-Madhu that's cool, but it won't work on 95% of the devices as they simply don't support per-key RGB in a first place :) It makes sense only for (some, not all) Strix laptops
